### PR TITLE
Update README and usage of various commands

### PR DESCRIPTION
**Pull request by carlevison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,57 +30,35 @@ Python 3.6
...
 ## Upload API
 
 Bindings for the Upload API.
 
+You can find documentation for each of the Upload API methods at [https://cloudinary.com/documentation/image_upload_api_reference](https://cloudinary.com/documentation/image_upload_api_reference) 
+
 The basic syntax using the Upload API is as follows:
 
 ```
 Usage: cld uploader [OPTIONS] [PARAMS]...
```
Comment (asisayag2) [inline README.md]: I think there's also a shortcut using "cld upload ... ". worth mentioning

Comment (asisayag2) [inline README.md]: Would add here the --help option

File: README.md [hunk 2]
```diff
@@ -4,7 +4,9 @@
 This command line interface is fully and seamlessly integrated with Cloudinary's APIs.
 
 ## Requirements
-Python 3.6
+Your own Cloudinary account.  If you don't already have one, sign up at [https://cloudinary.com/users/register/free](https://cloudinary.com/users/register/free)
+
+Python 3.6 or later
 
 ## Setup
```
[the remark this answers is not in the document]
Comment (carlevison) [reply README.md]: We seem to have the installation and configuration in one section here.  Is it OK to call it "Setup and Installation"?

File: README.md [hunk 3]
```diff
@@ -4,7 +4,9 @@
 This command line interface is fully and seamlessly integrated with Cloudinary's APIs.
 
 ## Requirements
```
[the remark this answers is not in the document]
Comment (carlevison) [reply README.md]: I'll point them at the official Python website.  pip is installed with Python now.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (asisayag2) [reply README.md]: Please consult @jackieros

File: README.md (hunk 1 above)
Comment (asisayag2) [reply README.md]: Ok

Comment (asisayag2) [reply README.md]: Yes, I think it's a good starting point for someone who doesn't want to read the documentation

**Pull request merged**